helm/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: p4pa-auth
+description: P4PA Auth
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+dependencies:
+  - name: microservice-chart
+    version: 5.8.0
```

Comment:
use the latest version:
https://github.com/pagopa/aks-microservice-chart-blueprint/releases